students/stuartkershaw/lesson03/strformat_lab.py

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env python3
+
+# Task One
+
+seq_in = (2, 123.4567, 10000, 12345.67)
+
+file_out = ''.join(['file_', format(seq_in[0], '03')])
+float_dec_out = format(seq_in[1], '.2f')
+sci_out = format(seq_in[2], '.2e')
+sci_sig_out = format(seq_in[3], '.2e')
+
+out_msg = '{}: {}, {}, {}'.format(file_out, float_dec_out, sci_out,
+                                  sci_sig_out)
```

Comment:
This could be written a bit more concisely and with a single call to `format()` like this:
```
print("file_{:03d} : {:10.2f}, {:.2e}, {:.3g}".format(2, 123.4567, 10000, 12345.67))
```